CHANGELOG.md

```diff
@@ -1,8 +1,12 @@
 # Braintree iOS SDK Release Notes
 
-## unreleased
+## unreleased (v7)
+* Breaking Changes
+  * Bump minimum supported deployment target to iOS 16+
 * BraintreePayPal
   * Add `BTPayPalRequest.userPhoneNumber` optional property
```

Comment:
nit: let's have an `unreleased (v7)` section at the top and an `unreleased` section after that so it's easier to quickly tell when we do releases what is a v7 only change. Will make it a bit easier to reason about when we need to merge main into v7.

Reply:
I think this CHANGELOG still combines the 2 unreleased sections and Jax is instead suggesting 2 sections

Reply:
ahh gotcha! thanks for clarifying y'all, addressed this in c386bf3f3eca0db14967a3da1611e2c0f6310230